packages/jss-plugin-compose/src/index.d.ts

```diff
@@ -1,3 +1,7 @@
 import {Plugin} from 'jss'
 
+export type ComposeProperties = {
+  composes: (string | string[])[]
```

Comment:
- possible values are a string or an array of strings 
- ideally if a string starts with "$", it should match with top level rule names (afaik it's possible now with TS)

Reply:
Template Literal Types is a good idea, but it is not working in this case because of string type.

For example:
```
`$${ClassName}' | string
```

This return always string.